metricbeat/docs/modules/aws.asciidoc

```diff
@@ -33,6 +33,9 @@ see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[Te
 aws> sts get-session-token --serial-number arn:aws:iam::1234:mfa/test@elastic.co --token-code 456789 --duration-seconds 129600
 ----
 
+Specific permissions needs to be added into the IAM user's policy to allow Metricbeat collecting AWS monitoring metrics. Please
```

Comment:
"to allow Metricbeat collecting" => "to authorize Metricbeat to collect", or something to that effect?

Reply:
And directly below, "metric set" => "metricset"?

Reply:
That was a left over which made it in this PR. Rebase will remove this part.

@kaiyan-sheng That is for you ^

Reply:
@webmat @ruflin Thanks for the comment. This is addressed in https://github.com/elastic/beats/pull/10334.